src/components/OrbSeal.tsx

```diff
@@ -8,8 +8,8 @@ export default function () {
   const { lg } = useBreakpoints()
   return (
     <svg
-      width={lg ? '471.6' : '262'}
-      height={lg ? '343.8' : '191'}
+      width={lg ? '125%' : '262'}
```

Comment:
should we use percents in non-`lg`, too?

Reply:
The defined non-lg sizing looks pretty good for all the non-lg screen sizes imo. We can change for consistency but I don't see an immediate need